lib/matplotlib/tests/test_backend_tk.py

```diff
@@ -18,30 +20,32 @@ def _isolated_tk_test(success_count, func=None):
 
     TkAgg tests seem to have interactions between tests, so isolate each test
     in a subprocess. See GH#18261
-
-    The decorated function must be fully self-contained, and thus perform
-    all the imports it needs.  Because its source is extracted and run by
-    itself, coverage will consider it as not being run, so it should be marked
-    with ``# pragma: no cover``
     """
 
     if func is None:
         return functools.partial(_isolated_tk_test, success_count)
 
-    # Remove decorators.
-    source = re.search(r"(?ms)^def .*", inspect.getsource(func)).group(0)
-
+    if "MPL_TEST_ESCAPE_HATCH" in os.environ:
```

Comment:
```suggestion
    if "MPL_TEST_ESCAPE_HATCH" in os.environ:
        # set in subprocess_run_helper() below
```
I was just wondering where that comes from